augur/data/schema-annotations.json

```diff
@@ -6,21 +6,51 @@
         "nuc": {
             "type": "object",
             "allOf": [{ "$ref": "#/$defs/startend" }],
+            "properties": {
+                "start": {
+                    "enum": [1],
+                    "$comment": "nuc must begin at 1"
+                },
+                "strand": {
+                    "type": "string",
+                    "not": {"enum":["-"]},
+                    "$comment": "The strand for 'nuc' is not needed, as it should be +ve for all genomes. Auspice will not proceed if you specify '-'"
+                }
```

Comment:
I guess we always emitted `"strand": "+"` previously so this is a backwards compatible change versus the alternative change of forbidding the `stand` property entirely?

This schema for strand seems weird though, it would allow _any_ string value as long as it's not `-`? Why not `"enum": ["+"]`?

Reply:
> Why not `"enum": ["+"]`

Done